nifi-nar-bundles/nifi-prometheus-bundle/nifi-prometheus-reporting-task/pom.xml

```diff
@@ -0,0 +1,134 @@
+
+	<dependencies>
+		<dependency>
+			<groupId>com.yammer.metrics</groupId>
+			<artifactId>metrics-core</artifactId>
+			<version>${yammer.metrics.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.nifi</groupId>
+			<artifactId>nifi-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.nifi</groupId>
+			<artifactId>nifi-utils</artifactId>
+ 			<version>1.10.0-SNAPSHOT</version>
+		</dependency>
+		<!-- The client -->
+		<dependency>
+			<groupId>io.prometheus</groupId>
+			<artifactId>simpleclient</artifactId>
+			<version>${prometheus.version}</version>
+		</dependency>
+		<!-- Hotspot JVM metrics -->
+		<dependency>
+			<groupId>io.prometheus</groupId>
+			<artifactId>simpleclient_hotspot</artifactId>
+			<version>${prometheus.version}</version>
+		</dependency>
+		<!-- Exposition servlet -->
+		<dependency>
+			<groupId>io.prometheus</groupId>
+			<artifactId>simpleclient_servlet</artifactId>
+			<version>${prometheus.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.nifi</groupId>
+			<artifactId>nifi-mock</artifactId>
+			<version>1.10.0-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+                <dependency>
+                        <groupId>org.apache.nifi</groupId>
+                        <artifactId>nifi-ssl-context-service</artifactId>
+                        <version>1.10.0-SNAPSHOT</version>
+                </dependency> 
+               <dependency>
+                       <groupId>org.apache.nifi</groupId>
+                       <artifactId>nifi-security-utils</artifactId>
+                       <version>1.10.0-SNAPSHOT</version>
+               </dependency>
+        
+              <dependency>
+                      <groupId>org.apache.nifi</groupId>
+                      <artifactId>nifi-ssl-context-service-api</artifactId>
+                      <version>1.10.0-SNAPSHOT</version>
+              </dependency>
+	      <dependency>
+		     <groupId>org.apache.nifi</groupId>
+		     <artifactId>nifi-standard-services-api-nar</artifactId>
+		     <version>1.10.0-SNAPSHOT</version>
+		     <type>nar</type>
+	      </dependency>
+	      <dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-continuation</artifactId>
+			<version>${jetty.version}</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-http</artifactId>
+			<version>${jetty.version}</version>
+			<scope>test</scope>
+		</dependency>
+
+
+		<dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-servlet</artifactId>
+			<version>${jetty.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.jetty</groupId>
+			<artifactId>jetty-server</artifactId>
+			<version>${jetty.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>javax.servlet</groupId>
+			<artifactId>javax.servlet-api</artifactId>
+			<version>3.0.1</version>
```

Comment:
do we not have a global javax servlet version set for the project? if so, we should use that. if not, we should use 3.1.0 over 3.0.1